### ISSUE TITLE
Cannot put wiki repo under organization.

### ISSUE THREAD

**Issue by bigdrum** [vsls-contrib/gistpad#268]
**Describe the bug**
Currently the wiki mode require the repo to be owned by the login user directly:
https://github.com/lostintangent/gistpad/blob/01a3ac2ae3a25f86d78eb09d7aeda3aeeba8ea97/src/repos/store/index.ts#L174

But I would like to have the wiki mode repo put under an organization. 

Is it possible to relax this restriction?

**Comment by bigdrum** [comment]
Probably the logic can be changed to:

If there is only one repo "isWiki" then use it regardless the name. The name condition is only used for more than one repo case.

**Comment by lostintangent** [comment]
Yep this proposal makes sense! I'll make this change in the next extension update (later this week).
